ui/admin/app/styles/app.scss

```diff
@@ -1047,3 +1047,14 @@
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+.remove-tag-modal {
```

Comment:
What is the purpose of adding styles this specific? 

Let me explain my question, I might have assume things. 
This modal looks like a confirmation modal, is that correct? if so, should we do it generic? or use one of the confirmation modals we already have?

I am fine if we have a new confirmation modal,but I would do it generic and not specific to `remove-tag`.

Reply:
I see what you mean here. I do think this is our only instance of a modal like this (needing to confirm before allowing the user to remove). Regardless, I see your point that it doesn't need to be specific to tags. I will update the class name.